components/ArticleList/ArticleList.styled.js

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+
+const StyledList = styled.ul`
+  margin: 0;
+  padding: 0;
+`;
```

Comment:
I recommend to rename file or the component, to be more consistent. 
And I guess you want/need  `list-style: none` 
```suggestion
const StyledArticleList = styled.ul`
list-style: none;
  margin: 0;
  padding: 0;
`;
```